core/metamodel/src/main/java/org/apache/isis/core/metamodel/facets/object/domainobjectlayout/CssClassFaFacetForDomainObjectLayoutAnnotation.java

```diff
@@ -0,0 +1,42 @@
+/* Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License. */
+package org.apache.isis.core.metamodel.facets.object.domainobjectlayout;
+
+import org.apache.isis.applib.annotation.CssClassFaPosition;
+import org.apache.isis.applib.annotation.DomainObjectLayout;
+import org.apache.isis.core.metamodel.facetapi.FacetHolder;
+import org.apache.isis.core.metamodel.facets.members.cssclassfa.CssClassFaFacet;
+import org.apache.isis.core.metamodel.facets.members.cssclassfa.CssClassFaFacetAbstract;
+
+import com.google.common.base.Strings;
+
+public class CssClassFaFacetForDomainObjectLayoutAnnotation extends CssClassFaFacetAbstract {
```

Comment:
Hi guys,

Coincidentally, I'm just merging this stuff in now.  Should be hitting
master in next half hour or so.

Cheers
Dan

On 23 April 2015 at 14:13, Martin Grigorov <notifications@github.com> wrote:

> In
> core/metamodel/src/main/java/org/apache/isis/core/metamodel/facets/object/domainobjectlayout/CssClassFaFacetForDomainObjectLayoutAnnotation.java
> <https://github.com/apache/isis/pull/29#discussion_r28960421>:
>
> > + * software distributed under the License is distributed on an
> > + * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
> > + * KIND, either express or implied. See the License for the
> > + * specific language governing permissions and limitations
> > + * under the License. */
> > +package org.apache.isis.core.metamodel.facets.object.domainobjectlayout;
> > +
> > +import org.apache.isis.applib.annotation.CssClassFaPosition;
> > +import org.apache.isis.applib.annotation.DomainObjectLayout;
> > +import org.apache.isis.core.metamodel.facetapi.FacetHolder;
> > +import org.apache.isis.core.metamodel.facets.members.cssclassfa.CssClassFaFacet;
> > +import org.apache.isis.core.metamodel.facets.members.cssclassfa.CssClassFaFacetAbstract;
> > +
> > +import com.google.common.base.Strings;
> > +
> > +public class CssClassFaFacetForDomainObjectLayoutAnnotation extends CssClassFaFacetAbstract {
>
> No. Leave it as is. Maybe there is a reason why there are both classes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/isis/pull/29/files#r28960421>.
>